framework/Models/HybridModels/HybridModelBase.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on April, 2020
+
+@author: wangc
+"""
+
+#External Modules------------------------------------------------------------------------------------
+import copy
+import abc
+import numpy as np
+from numpy import linalg
+import time
+import itertools
+from collections import OrderedDict
```

Comment:
Are all these imports used? It seems like `linalg` might not be, maybe others.

Reply:
removed.